packages/upload-client/src/store.js

```diff
@@ -128,6 +127,16 @@ export async function add(
     }
   )
 
+  if (!fetchDidCallUploadProgressCb && options.onUploadProgress) {
+    // the fetch implementation didn't support onUploadProgress
+    const carBlob = new Blob([car])
+    options.onUploadProgress({
+      total: carBlob.size,
```

Comment:
are `total` and `loaded` here the same because this will be the only progress event sent?

Reply:
yep! the only progress even for this car file, and all we know here is the fetch succeeded so it's 100% progress all at once